samples/client/petstore/elixir/lib/openapi_petstore/api/fake.ex

```diff
@@ -180,14 +234,14 @@ defmodule OpenapiPetstore.Api.Fake do
 
   - `connection` (OpenapiPetstore.Connection): Connection to server
   - `opts` (keyword): Optional parameters
-    - `:body` (float()): Input number as post body
+    - `:body` (number()): Input number as post body
```

Comment:
`number()` is more accurate since [we are not specifying any format in the OAS](https://github.com/OpenAPITools/openapi-generator/blob/23e74d7b79fb6c7b01e5bfc8548440e1952f24e8/modules/openapi-generator/src/test/resources/3_0/elixir/petstore-with-fake-endpoints-models-for-testing.yaml#L1975). `number()` captures whole numbers and floating point numbers (see [here](https://hexdocs.pm/elixir/1.18.3/typespecs.html#built-in-types)).